tests/integration/dbapi/async/test_queries_async.py

```diff
@@ -353,3 +355,46 @@ async def test_set_invalid_parameter(connection: Connection):
             await c.execute("set some_invalid_parameter = 1")
 
         assert len(c._set_parameters) == 0
+
+
+@mark.asyncio
+async def test_ss_async_execution_query(connection: Connection) -> None:
+    """Make an sql query and receive an id back."""
+    with connection.cursor() as c:
+        query_id = await c.execute(
+            "DROP TABLE IF EXISTS test_tb", [], async_execution=True
```

Comment:
Make sure test_db exists

Reply:
I changed it to `SELECT 1`. All I care about is getting a query_id back.